src/App/Webhook/PdkWebhookManager.php

```diff
@@ -123,7 +117,16 @@ protected function resolveHook(string $event): HookInterface
      */
     private function getHooks(Request $request): array
     {
-        $body = json_decode($request->getContent(), true);
+        $headers = $request->headers->all();
+        $body    = json_decode($request->getContent(), true);
+
+        if (isset($headers['x-myparcel-hook'])) {
+            $body['data']['hooks'] = array_map(function ($hook) use ($headers) {
```

Comment:
Het is denk ik mooier om het resultaat van deze map in een losse variabele te zetten (bijvoorbeeld $hooks ofzo) ipv de body te modificeren en vervolgens te returnen